odbc/src/api/functions.rs

```diff
@@ -850,8 +848,7 @@ fn sql_driver_connect(conn: &Connection, odbc_uri_string: &str) -> Result<MongoC
     // create an impl From Result<T, mongo_odbc_core::Error> to Result<T, ODBCError>
     // hence this bizarre Ok(func?) pattern.
     Ok(mongo_odbc_core::MongoConnection::connect(
-        &mongo_uri,
-        auth_src,
+        client_options,
```

Comment:
Can you also remove this line above since app_name is handled in the URI:
`let application_name = odbc_uri.remove(&["app_name", "application_name"]);`

Reply:
Do we want to remove the app_name option then?

Reply:
Ahh never mind.